docs-src/concepts/what-is-a-workflow-id-reuse-policy.md

```diff
@@ -33,7 +33,9 @@ The Workflow Id Reuse Policy can have one of the following values:
   Use this when there can only be one Workflow Execution per Workflow Id within a Namespace for the given retention period.
 - **Terminate if Running:** Specifies that if a Workflow Execution with the same Workflow Id is already running, it should be terminated and a new Workflow Execution with the same Workflow Id should be started. This policy allows for only one Workflow Execution with a specific Workflow Id to be running at any given time.
 
-A Workflow Id Reuse Policy applies only if a Closed Workflow Execution with the same Workflow Id exists within the Retention Period of the associated Namespace.
-For example, if the Namespace's retention period is 30 days, a Workflow Id Reuse Policy can only compare the Workflow Id of the spawning Workflow Execution against the Closed Workflow Executions for the last 30 days.
+The first three values (Allow Duplicate, Allow Duplicate Failed Only, and Reject Duplicate) of the Workflow Id Reuse Policy apply to Closed Workflow Executions that are retained within the Namespace.
+For example, the Temporal Server can only check the Workflow Id of the spawning Workflow Execution based on the Workflow Id Reuse Policy against the Closed Workflow Executions for the last 30 days given a default Retention Period.
```

Comment:
```suggestion
For example, the Temporal Service can only check the Workflow Id of the spawning Workflow Execution based on the Workflow Id Reuse Policy against the Closed Workflow Executions for the last 30 days given a default Retention Period.
```

Let's use the generic "Temporal Service" here so that it can apply to Cloud as well.
I apologize for not calling this out sooner.
We just agreed on using this term recently, but should start using it asap.